package.json

```diff
@@ -40,15 +40,24 @@
     "@mantine/next": "^5.10.4",
     "@mantine/notifications": "^5.10.4",
     "@mantine/spotlight": "^5.10.4",
+    "@nivo/bump": "^0.80.0",
+    "@nivo/core": "^0.80.0",
+    "@nivo/line": "^0.80.0",
```

Comment:
We can remove hte nivo if we don't usr eit 👍

Reply:
If nobody else needs it :) I removed it from my package.json.